### PR TITLE
Printable protocol summary

### PR DESCRIPTION
**Pull request by wwqrd**
Adds the ability to preview and then print a summary of the open protocol.

### CODE REVIEW

**Comment by jthrilly** [comment]
This seems to have a couple of missing features:

- [ ] Add a "Print protocol summary" button to the codebook view
- [ ] Restyle the preview window as a print preview (Electron doesn't support the Chrome print preview functionality)
- [ ] Make the filename reflect the protocol name

File: package.json [hunk 1]
```diff
@@ -263,6 +265,7 @@
         }
     },
     "dependencies": {
-        "postcss": "^8.2.8"
+        "postcss": "^8.2.8",
+        "react-router-dom": "~5.2.0"
```
Comment (jthrilly) [inline package.json:269]: Should this be a dev dependency?

File: src/ducks/store.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import ipc from './ipc';
 import { rootReducer } from './modules/root';
 
 const persistConfig = {
-  key: 'architect',
+  key: 'architect6.1',
```
Comment (jthrilly) [inline src/ducks/store.js:10]: Why was this changed? This will reset all app state for current users, won't it? We should revert unless it was intentional.

File: src/lib/ProtocolSummary/ProtocolSummary.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.printable-summary {
```
Comment (jthrilly) [inline src/lib/ProtocolSummary/ProtocolSummary.scss:1]: Sorry about this - needs to be removed.

File: src/lib/ProtocolSummary/components/Stage/Presets.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  // eslint-disable-next-line react/forbid-prop-types
+  presets: PropTypes.object,
+};
+
+Presets.defaultProps = {
+  presets: null,
+};
+
+export default Presets;
+// edges: {display: Array(1)}
```
Comment (jthrilly) [inline src/lib/ProtocolSummary/components/Stage/Presets.js:72]: Remove

**Review by jthrilly — COMMENTED** [review]
In addition to the issues I highlighted, we also need to fix the proptype console errors in the app.

File: src/ducks/store.js (hunk 2 above)
Comment (wwqrd) [reply src/ducks/store.js:10]: Agreed, this shouldn't change unless the store has changed in some other drastic way (doesn't look like it has to me).
